packages/components/organisms/f-header/src/components/_tests/Logo.test.js

```diff
@@ -47,6 +47,23 @@ describe('Logo', () => {
         expect(logo).toBeDefined();
     });
 
+    it('should render Jet logo if jet theme passed', () => {
+        // Arrange
+        const propsData = {
+            theme: 'jet',
```

Comment:
Instead of setting the theme i think this test should be passing the prop value `useJetLogo` as `true` instead and then the theme (computed) would automatically allow this test to pass.

Reply:
Because this is the test for logo component it doesn't know about `shouldUseJetLogo` prop as it a header component's prop. I added another test to the header.test.js to cover that prop.